libraries/core_libs/network/include/network/tarcap/packets_handlers/get_dag_sync_packet_handler.hpp

```diff
@@ -11,12 +12,14 @@ class TransactionManager;
 
 namespace taraxa::network::tarcap {
 
+class SyncingState;
```

Comment:
is there packet handler that does not use `SyncingState` ? if there is non or just one or two, maybe it should be part of `PacketHandler` class

Reply:
you can see it in registerHandlers function, I think 7 out of 11 are now using SyncingState but they use syncing state only because they need to access malicious_peers, which could be part of PacketHandler rather than syncing_state - check out my first comment

Reply:
Moved to PeersState